src/alert_device.cc

```diff
@@ -106,6 +121,16 @@ Device::addAlert (const std::string& quantity, const std::map<std::string,std::v
     ) {
         log_error ("aa: thresholds for %s are not present in %s", quantity.c_str (), assetName ().c_str ());
     } else {
+        alert.ruleRescanned = true;
+        if (updatingalert && alert.rulePublished) {
+            // If anything changed, reset the flag to make the info known
+            if (alert.lowWarning != existingalert.lowWarning)     alert.rulePublished = false;
```

Comment:
Reminder to self: this code passed a long and turbulent evolution that day, and the stack of `if`'s was easier to step through in a debugger. If this small bit of performance helps later, it can be collapsed into one `if` with a bigger condition, so any one hit suffices to flip the flag.